onnxruntime/python/tools/transformers/models/stable_diffusion/README.md

```diff
@@ -4,7 +4,7 @@
 
 [Stable Diffusion](https://stability.ai/blog/stable-diffusion-announcement) is a text-to-image latent diffusion model for image generation. Explanation of the Stable Diffusion can be found in [Stable Diffusion with Diffusers](https://huggingface.co/blog/stable_diffusion).
 
-## CUDA Optimizations for Stable Diffusion
+## Optimizations for Stable Diffusion
```

Comment:
Suggest changing the title as well: 
Stable Diffusion CUDA Optimization => Stable Diffusion GPU Optimization